api/src/feeds/impl/feeds_api_impl.py

```diff
@@ -12,6 +16,24 @@
 from feeds_gen.models.latest_dataset import LatestDataset
 from feeds_gen.models.source_info import SourceInfo
 
+from database.database import Database
+
+db = Database()
+
+
+def map_feed(feed: Feed):
```

Comment:
Suggestion: this function could be added to a `utils` file/module if you think it's going to be used by other classes or added as a static method of the `FeedApiImpl` class otherwise to keep the coding style consistent.

Reply:
moved to a static method in this class as this looks specific to this module